src/algebra/category/CommRing/limits.lean

```diff
@@ -120,6 +121,9 @@ begin
     erw (s.π.app j).map_add, refl }
 end
 
+end CommRing_has_limits
```

Comment:
```suggestion
end CommRing
```

Reply:
I can't commit this suggestion. Maybe github is being buggy...